src/platform/plugins/shared/discover/public/__mocks__/services.ts

```diff
@@ -174,10 +174,10 @@ export function createDiscoverServicesMock(): DiscoverServices {
     docLinks: docLinksServiceMock.createStartContract(),
     embeddable: embeddablePluginMock.createStartContract(),
     capabilities: {
-      visualize: {
+      visualize_v2: {
         show: true,
       },
-      discover: {
+      discover_v2: {
```

Comment:
We have some functional test suits for controlling privileges to Saved queries. Should not we revisit them? Interesting that they keep passing as is.

x-pack/test/functional/apps/saved_query_management/feature_controls/security.ts
x-pack/test/functional/apps/discover/feature_controls/discover_security.ts

Utils: x-pack/test/functional/apps/saved_query_management/utils/saved_query_security.ts

Reply:
I actually think it's better if we leave them as they are since currently the roles are created with V1 privileges, so the fact that they still pass helps validate that both the V1 -> V2 migration is working as intended, and that the new handling of the `savedQueryManagement` privilege is also working. We could add something additional if we feel it's necessary to have explicit V2 tests, but personally I think these tests already do a good job of validating the expected behaviour.

Reply:
Alright, thanks!